tests/cpp/filesystem.h

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) 2022 by XGBoost Contributors
+ */
+#pragma once
```

Comment:
Please use include guard macros instead. CPPLint requires it.

Reply:
Changed to using macro guard.